Source/Core/CircleGeometry.js

```diff
@@ -175,5 +175,12 @@ define([
         });
     };
 
+    /**
+     * @private
```

Comment:
1. It looks like all surface geometries must implement `getRectangle`, so why is it private?  This means it's impossible for end users to implement custom surface geometry without use of the private API, doesn't it?